lib/internal/bootstrap/node.js

```diff
@@ -693,7 +693,9 @@ function setupProcessFatal() {
 }
 
 function wrapForBreakOnFirstLine(source) {
-  if (!process._breakFirstLine)
+  const { getOptionValue } = NativeModule.require('internal/options');
+  if (!(getOptionValue('--inspect-brk') ||
+      getOptionValue('--inspect-brk-node')))
```

Comment:
I think only `!(getOptionValue('--inspect-brk')` is necessary here? We don't really support `-e --inspect-brk-node` at the moment anyways.